test-apps/display-performance-test-app/.env

```diff
@@ -0,0 +1,9 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Can this be removed now that we updated `eslint`?

Reply:
I tried that, but still ran into issues. For some reason the global eslint ver in the monorepo is still 6.8.0. Not sure why the lock file didnt get updated to 7.21.0. I didn't want to manually play with the lock file, and since we're disabling eslint i thought it was fine. Any idea how to clean this up in rush?

![image](https://user-images.githubusercontent.com/11051042/110844478-76c0d480-8277-11eb-887a-432ca0ff68bd.png)

Reply:
Strange... I'll pull and see what's going on.